components/core/Virtualization/IVirtualizeJsCallbacks.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if NET5_0 || NET6_0
```

Comment:
Wouldn't it be more future-proof if it was `NET5_0_OR_GRATER`?

Reply:
I played around with what you wrote and at first I could not recreate your findings. What I found out is that you have to have a razor file for this to actually behave as expected.
So, if following this example:
1. Having only `ComponentForNet5.cs` will create an element, as in the screenshot.
2. Having only `ComponentForNet5.razor.cs` will also create an element.
3. Having `ComponentForNet5.razor` and `ComponentForNet5.razor.cs` does not produce the element.
I do not know if you want to make appropriate changes, but at least this is some kind of an explanation.